Swift.sublime-syntax

```diff
@@ -344,10 +344,10 @@ contexts:
       scope: keyword.operator
     - match: \w+
       scope: support.class
-    - match: '>'
+    - match: '>\s*'
       pop: true
   class_type:
-    - match: \w+
+    - match: \w+\s*
```

Comment:
I put this in `syntax_test.swift`:
![image](https://cloud.githubusercontent.com/assets/7380931/14621965/09d2537e-060a-11e6-9b53-75e038b2c27b.png)

I'm on dev build 3111. And I have the latest of the syntax file from package control, which is why the errors started appearing :)